install/install.sh

```
@@ -0,0 +1,521 @@
+    if [[ -z "$latest_version" ]]; then
+        print_error "Failed to fetch latest version"
+        exit 1
+    fi
+    
+    echo "$latest_version"
+}
+
+# Function to calculate hash
+calculate_hash() {
+    local file="$1"
+    local platform=$(detect_platform)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)